doc/tutorial/Docs.lhs

```diff
@@ -220,12 +220,10 @@ api :: Proxy DocsAPI
 api = Proxy
 
 server :: Server DocsAPI
-server = Server.server3 :<|> serveDocs
-
-  where serveDocs _ respond =
-          respond $ responseLBS ok200 [plain] docsBS
-
-        plain = ("Content-Type", "text/plain")
+server = Server.server3 :<|> Tagged serveDocs where
```

Comment:
I guess it's price to pay; the `type family Tag m a where Tag m a = a` won't work, as it will reduce too early, so `Enter` couldn't match it (and we cannot have type family application in the LHS of `Entered` or in `instance` declaration).

Now as I think, the `Entered` is probably not required; yet it might help generate better error messages, I should test a bit.